doc/project/release_process.rst

```diff
@@ -331,6 +362,50 @@ Processes to achieve selected certification will be determined by the Security a
 Safety Working Groups and coordinated with the TSC.
 
 
+Hardware Support Tiers
+***********************
+
+Tier 0: Emulation Platforms
+===========================
+
+- Tests are both built and run in these platforms in CI, and therefore runtime
+  failures can block Pull Requests.
+- Supported by the Zephyr project itself, commitment to fix bugs in releases.
+- One Tier 0 platform is required for each new architecture.
+- Bugs reported against platforms of this tier are to be evaluated and treated as
+  a general bug in Zephyr and should be dealt with the highest priority.
+
+Tier 1: Supported Platforms
+===========================
+
+- Commitment from a specific team to run tests using twister device
```

Comment:
Related to my comment above, what does "specific team" mean? Working Group or is this a community member's HW range?

Reply:
a team behind a platform, i.e. a vendor.